tests/Integration/Behaviour/Features/Scenario/Product/update_status.feature

```diff
@@ -64,3 +69,26 @@ Feature: Update product status from BO (Back Office)
     When I enable product "product1"
     Then product "product1" should be enabled
     And product "product1" should be indexed
+
+  Scenario: I can not publish a product without a name
+    When I add product "product4" with following information:
+      | type        | standard       |
+    Then product "product4" should be disabled
+    And product "product4" type should be standard
+    And product "product4" localized "name" should be:
+      | locale | value |
+      | en-US  |       |
```

Comment:
maybe it would be cleaner to have a `Then product "product 4" should have no name" rather than checking an empty string on a specific language

Reply:
I could add a new end point indeed, but this one was already available and performs the same thing Besides in this case I prefer the fact that the test reflects the fact that we are checking the empty value for a specific language (the default one)

Reply:
ok 👍